specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2023-12-15/examples/AzureIaasVm/BackupCrrJob_Cancel.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "azureRegion": "southeastasia",
+    "api-version": "2023-12-15",
+    "parameters": {
+      "resourceId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/testRg/providers/Microsoft.RecoveryServices/vaults/testVault",
+      "jobName": "02585cc9-d7f4-4b46-860c-14c048cce178"
```

Comment:
>jobName [](http://example.com/codeflow?start=7&length=7)

would it make more sense for this to be an uri parameter?

Reply:
Here the scenario is Cross Region Restore where we are restoring for the customer from another region, we didnt add job/{jobId}/cancel because job/{jobId} is not a resource present in the target region, hence we didnt find it logical to add this proxy resource for the target region. Also the scenario in this PR is cancelling a cross region restore operation and we just wanted to add in lines of the existing behaviour